src/spyglass/decoding/v1/core.py

```diff
@@ -22,26 +28,78 @@ class DecodingParameters(SpyglassMixin, dj.Lookup):
     decoding_kwargs : BLOB             # additional keyword arguments
     """
 
-    # contents = [
-    #     [
-    #         "contfrag_clusterless",
-    #         vars(ContFragClusterlessClassifier()),
-    #         dict(),
-    #     ],
-    # ]
+    contents = [
+        {
+            "decoding_param_name": "contfrag_clusterless",
+            "decoding_params": ContFragClusterlessClassifier(),
+            "decoding_kwargs": dict(),
+        },
+        {
+            "decoding_param_name": "nonlocal_clusterless",
+            "decoding_params": NonLocalClusterlessDetector(),
+            "decoding_kwargs": dict(),
+        },
+        {
+            "decoding_param_name": "contfrag_sorted",
+            "decoding_params": ContFragSortedSpikesClassifier(),
+            "decoding_kwargs": dict(),
+        },
+        {
+            "decoding_param_name": "nonlocal_sorted",
+            "decoding_params": NonLocalSortedSpikesDetector(),
+            "decoding_kwargs": dict(),
+        },
+    ]
 
-    # @classmethod
-    # def insert_default(cls):
-    #     cls.insert(cls.contents, skip_duplicates=True)
+    @classmethod
+    def insert_default(cls):
+        cls.insert(cls.contents, skip_duplicates=True)
 
-    # def insert(self, keys, **kwargs):
-    #     pass
+    def insert(
+        self,
+        rows,
+        replace=False,
+        skip_duplicates=False,
+        ignore_extra_fields=False,
+        allow_direct_insert=None,
```

Comment:
I think you could replace any params you're passing to `super` with `*args, **kwargs`